services/navigation/builtin/builtin_test.go

```diff
@@ -83,6 +83,52 @@ func deleteAllWaypoints(ctx context.Context, svc navigation.Service) error {
 	return nil
 }
 
+func TestNavExploreMode(t *testing.T) {
+	ctx := context.Background()
+
+	cases := []struct {
+		description string
+		mode        navigation.Mode
+		expectedErr error
+	}{
+		{
+			description: "setting mode to manual",
+			mode:        navigation.ModeManual,
+			expectedErr: nil,
+		},
+		{
+			description: "setting mode to waypoint",
+			mode:        navigation.ModeManual,
```

Comment:
ModeWaypoint?

Reply:
You are correct, it has now been fixed, apologies

Reply:
Looks like this still needs to be changed to ModeWaypoint